copilot/fsd-application-store/manifest.yml

```diff
@@ -50,7 +50,8 @@ variables:
   PRIMARY_QUEUE_URL:
     from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-AssessmentImportQueueURL
   DEAD_LETTER_QUEUE_URL:
-    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-DeadLetterQueueURL
+    from_cfn: ${COPILOT_APPLICATION_NAME}-${COPILOT_ENVIRONMENT_NAME}-AssessmentImportDeadLetterQueueURL
```

Comment:
To match with the change made at the workflow level: https://github.com/communitiesuk/funding-service-design-workflows/blob/main/apps/pre-award/copilot/environments/addons/assessment-import-queue.yml#L60